front-packages/invite-user/src/feature/InviteUserApp.tsx

```diff
@@ -99,7 +104,7 @@ const InviteUserApp = () => {
         </Table>
         {invitedUsers && invitedUsers.length === 0 && (
           <IllustrationContainer>
-            <SurveyIllustration />
+            <SurveyIllustrationContainer />
```

Comment:
You could the props `size` instead: 
```suggestion
            <SurveyIllustration size={128} />
```